### PR TITLE
Lock py-evm==0.5.0a3

### PR DESCRIPTION
**Pull request by Uxio0**
- Fix issues with dependency resolving

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -11,8 +11,7 @@
 requirements = [
     "cached-property>=1.5; python_version < '3.8'",
     "eip712_structs",
-    "packaging",
```
Comment (fmrsabino) [inline setup.py:14]: So this was not required?

File: gnosis/eth/constants.py [hunk 2]
```diff
@@ -1,4 +1,6 @@
-from eth.constants import SECPK1_N
+SECPK1_N = (
+    115792089237316195423570985008687907852837564279074904382605163141518161494337
+)
```
Comment (fmrsabino) [inline gnosis/eth/constants.py:3]: `py-evm.usages--` 😛

**Review by fmrsabino — APPROVED** [review]

File: setup.py (hunk 1 above)
Comment (Uxio0) [reply setup.py:14]: It should be included in any django distrution, but we are using it, I will put it back

Comment (Uxio0) [reply setup.py:14]: Done

**Pull request merged**